libs/designsystem/src/lib/components/form-field/form-field.component.ts

```diff
@@ -1,45 +1,67 @@
 import {
+  AfterContentChecked,
+  AfterContentInit,
   ChangeDetectionStrategy,
   Component,
-  Input,
   ContentChild,
-  AfterContentChecked,
-  OnDestroy,
   ElementRef,
-  OnInit,
   HostListener,
+  Input,
+  OnDestroy,
+  OnInit,
+  Renderer2,
 } from '@angular/core';
 
 import { PlatformService } from '../../helpers/platform.service';
 import { WindowRef } from '../../types/window-ref';
+import { RadioGroupComponent } from '../radio';
 
 import { InputCounterComponent } from './input-counter/input-counter.component';
+import { InputComponent } from './input/input.component';
+import { TextareaComponent } from './textarea/textarea.component';
+
+// Counter for generating unique element ids
+let uniqueId = 0;
```

Comment:
Consider a simple generator function instead

Reply:
@jb-roa Why?

Reply:
To express what you are actually doing, generating a sequence of values. For improved readability, i.e.:
`_labelId = 'kirby-form-field-label-${uniqueId.next()}';`

Reply:
Ok check, good point - added a UniqueIdGenerator class 👍